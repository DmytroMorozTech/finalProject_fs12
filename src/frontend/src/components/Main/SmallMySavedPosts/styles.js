import { makeStyles } from '@material-ui/core/styles'

export default makeStyles((theme) => ({

  smallMyItems: {
    display: 'flex',
    justifyContent: 'space-between',
    border: theme.border.simple,
    borderRadius: theme.shape.medium,
    backgroundColor: theme.palette.common.white,
    padding: theme.spacing(4),
    marginTop: theme.spacing(4),
    width: '100%',
    height: '70px',
    textDecoration: 'none',
    color: theme.palette.primary.main,
    fontWeight: theme.typography.fontWeightBold,
    fontSize: theme.typography.h5.fontSize,
    cursor: 'pointer'
  },

  item: {
    display: 'flex',
    alignItems: 'center'
  },

  icon: {
    fontSize: theme.typography.icons.medium.fontSize
  },

  title: {
    fontSize: theme.typography.h5.fontSize,
    paddingLeft: theme.spacing(2)
  }
}))
