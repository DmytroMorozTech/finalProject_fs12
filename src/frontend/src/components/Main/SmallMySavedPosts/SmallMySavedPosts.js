import React from 'react'
import styles from './styles'
import { Link } from 'react-router-dom'
import BookmarkIcon from '@material-ui/icons/Bookmark'

function SmallMySavedPosts () {
  const classes = styles()

  return (
    <Link to='/bookmarked' className={classes.smallMyItems}>
      <div className={classes.item}>
        <BookmarkIcon fontSize="inherit" className={classes.icon}/>
        <span className={classes.title}>My saved posts</span>
      </div>
    </Link>
  )
}

export default SmallMySavedPosts
