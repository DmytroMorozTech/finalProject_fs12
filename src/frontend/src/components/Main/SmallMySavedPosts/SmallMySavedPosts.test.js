import { shallow } from 'enzyme'
import React from 'react'
import SmallMySavedPosts from './SmallMySavedPosts'

jest.mock('./styles', () => () => ({}))

const setUp = () => shallow(<SmallMySavedPosts />)

describe('should render SmallMySavedPosts component', () => {
  let component
  beforeEach(() => {
    component = setUp()
  })
  it('to match snapshot', () => {
    expect(component).toMatchSnapshot()
  })
})
