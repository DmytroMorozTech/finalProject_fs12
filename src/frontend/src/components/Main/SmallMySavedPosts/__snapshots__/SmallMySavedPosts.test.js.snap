// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SmallMySavedPosts component to match snapshot 1`] = `
<Link
  to="/bookmarked"
>
  <div>
    <Memo(ForwardRef(BookmarkIcon))
      fontSize="inherit"
    />
    <span>
      My saved posts
    </span>
  </div>
</Link>
`;
